src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
```suggestion
public class Application {
```
application should not extdent functionality and properties of Lottery